go.mod

```diff
@@ -12,20 +12,21 @@ require (
 	github.com/spf13/cobra v1.7.0
 	github.com/stretchr/testify v1.8.4
 	go.opencensus.io v0.24.0
-	go.opentelemetry.io/collector/component v0.79.0
-	go.opentelemetry.io/collector/config/confignet v0.0.0-20230619134516-d8100bb80fe1
-	go.opentelemetry.io/collector/config/configtelemetry v0.0.0-20230619134516-d8100bb80fe1
-	go.opentelemetry.io/collector/confmap v0.79.0
-	go.opentelemetry.io/collector/connector v0.0.0-20230615165320-df20186ee21c
-	go.opentelemetry.io/collector/consumer v0.79.0
-	go.opentelemetry.io/collector/exporter v0.79.0
-	go.opentelemetry.io/collector/extension v0.0.0-20230609200026-525adf4a682a
-	go.opentelemetry.io/collector/extension/zpagesextension v0.79.0
-	go.opentelemetry.io/collector/featuregate v1.0.0-rcv0012
-	go.opentelemetry.io/collector/pdata v1.0.0-rcv0012
-	go.opentelemetry.io/collector/processor v0.0.0-20230609193203-89d1060c7606
-	go.opentelemetry.io/collector/receiver v0.79.0
-	go.opentelemetry.io/collector/semconv v0.79.0
+	go.opentelemetry.io/collector v0.80.0
```

Comment:
Not sure it will work, due to this version change. Basically, between the time when @xvilaca-te made his changes and this PR, some contributor made a refactoring which requires not yet released version